File: tools/qvod/src/test/java/top/bettercode/summer/tools/qvod/test/QControllerTest.java
package top.bettercode.summer.tools.qvod.test;

import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;

import org.junit.jupiter.api.Test;
import top.bettercode.summer.test.BaseWebNoAuthTest;

public class QControllerTest extends BaseWebNoAuthTest {

  @Test
  public void test() throws Exception {
    mockMvc.perform(post("/test")
    ).andExpect(status().isOk());
  }
}

File: tools/qvod/src/test/kotlin/top/bettercode/summer/tools/qvod/test/QControllerTest.kt
package top.bettercode.summer.tools.qvod.test

import org.junit.jupiter.api.Test
import org.springframework.test.web.servlet.result.MockMvcResultMatchers
import top.bettercode.summer.test.BaseWebNoAuthTest


class QControllerTest : BaseWebNoAuthTest() {
    @Test
    fun test() {
        mockMvc.perform(post("/test")
        ).andExpect(MockMvcResultMatchers.status().isOk())
    }
}
